CONTRIBUTING.md

```diff
@@ -0,0 +1,144 @@
+
+## Project governance
+
+The Ladybird project is driven by the [Ladybird Browser Initiative](https://ladybird.org/), a 501(c)(3) non-profit organization.
+
+Pull requests are approved and merged by a group of maintainers. The current group of maintainers are (in alphabetical order):
+
+- [Ali Mohammad Pur (@alimpfard)](https://github.com/alimpfard)
+- [Aliaksandr Kalenik (@kalenikaliaksandr)](https://github.com/kalenikaliaksandr)
+- [Andreas Kling (@awesomekling)](https://github.com/awesomekling)
+- [Andrew Kaster (@ADKaster)](https://github.com/ADKaster)
+- [Dániel Bertalan (@BertalanD)](https://github.com/BertalanD)
+- [Jelle Raaijmakers (@GMTA)](https://github.com/gmta)
+- [Luke Wilde (@Lubrsi)](https://github.com/Lubrsi)
+- [Sam Atkins (@AtkinsSJ)](https://github.com/AtkinsSJ)
+- [Shannon Booth (@shannonbooth)](https://github.com/shannonbooth)
+- [Tim Flynn (@trflynn89)](https://github.com/trflynn89)
+- [Tim Ledbetter (@tcl3)](https://github.com/tcl3)
+
+## Human language policy
+
+In Ladybird, we treat human language as seriously as we do programming language.
+
+The following applies to all user-facing strings, code, comments, and commit messages:
+
+* The official project language is American English with ISO 8601 dates and metric units.
+* Use proper spelling, grammar, and punctuation.
+* Write in an authoritative and technical tone.
+* Avoid contractions, slang, and idioms.
+* Avoid humor, sarcasm, and other forms of non-literal language.
+* Use gender-neutral pronouns, except when referring to a specific person.
+
+Note that this also applies to debug logging and other internal strings, as they may be exposed to users in the future.
+
+## Code submission policy
+
+Nobody is perfect, and sometimes we mess things up. That said, here are some good do's & don'ts to try and stick to:
+
+**Do:**
+
+* Choose expressive variable, function and class names. Make it as obvious as possible what the code is doing.
+* Split your changes into separate, atomic commits (i.e. A commit per feature or fix, where the build, tests and the system are all functioning).
+* Make sure your commits are rebased on the master branch.
+* Wrap your commit messages at 72 characters.
+* The first line of the commit message is the subject line, and must have the format "Category: Brief description of what's being changed". The category should be the name of a library, application, service, utility, etc.
+  * Examples: `LibMedia`, `WebContent`, `CI`, `AK`, `RequestServer`, `js`
+  * Don't use a category like "`Libraries`" or "`Utilities`", except for generic changes that affect a large portion of code within these directories.
+  * Don't use specific component names, e.g. C++ class names, as the category either - mention them in the summary instead. E.g. `LibGUI: Brief description of what's being changed in FooWidget` rather than `FooWidget: Brief description of what's being changed`
+  * Several categories may be combined with `+`, e.g. `LibJS+LibWeb+Browser: ...`
```

Comment:
I deleted most of the C++-isms from this section, but kept this because thinking of new website-specific component examples didn't sound like fun.